.github/scripts/eval_opencompass_config.py

```diff
@@ -46,6 +44,17 @@
 ],
                             eos_token_id=2)
 
+llama3_meta_template = dict(round=[
+    dict(role='HUMAN',
+         begin='<|start_header_id|>user<|end_header_id|>\n\n',
+         end='<|eot_id|>'),
+    dict(role='BOT',
+         begin='<|start_header_id|>assistant<|end_header_id|>\n\n',
+         end='<|eot_id|>',
+         generate=True),
+],
+                            eos_token_id=[128001, 128009])
```

Comment:
Inappropriate indent, please reformat it.

Reply:
> Inappropriate indent, please reformat it.

It's reformatted by pre-commit like this. I think it's ok.